k8s/custom-governance/README.md

```diff
@@ -76,7 +58,7 @@ Questions about Custom Governance? Go [here](https://docs.google.com/document/d/
 
 ## Before you get started
 
-As of fall 2020 the Custom Governance product is in Beta stage with allowlisting. We are taking a limited number of new customers. To try out Custom Governance, [**please fill out this intake form to be allowlisted.**](https://docs.google.com/forms/d/1ulI20NPs-S5-pAAZDSlZl4nAOhg5GRERtyoAr0XM6lg/viewform?edit_requested=true)
+As of winter 2020 the Custom Governance product is in Beta stage with allowlisting. We are taking a limited number of new customers. To try out Custom Governance, [**please fill out this intake form to be allowlisted.**](https://docs.google.com/forms/d/e/1FAIpQLSejlXV2IJbH7IlUGAHDkpNc_-UFPw4oB3K1TccbuBCwvuTVwA/viewform?gxids=7826)
```

Comment:
We should use Early Access Center URL going forward.

Reply:
Can you point me out what is the Early Access Center URL?

Reply:
https://console.cloud.google.com/iam-admin/early-access-center/campaigns/87175a3a-c0ae-4551-a6f6-b586a45bdcf9/enroll-customer should allow customers to enroll, or if they are pre-invited they can go directly to https://console.cloud.google.com/iam-admin/early-access-center. I think we could also keep link to the form as a backup.